src/controllers/GenericResponse.js

```diff
@@ -0,0 +1,33 @@
+import Api from '../api/Api';
```

Comment:
Should this file be in Controllers package, or move to some utils/filters/some logical package?

Reply:
It has been moved to the utilities package